app/views/pages/hiring_content.html.haml

```diff
@@ -0,0 +1,145 @@
+#page-hiring-job.bg-backgroundColorA.pb-16
+  %header.c-shapes
+    .md-container
+      = link_to hiring_page_path, class: "c-prominent-link" do
+        = graphical_icon "arrow-left"
+        Back to All Job Listings
+
+      .role
+        = graphical_icon "hiring-fundraiser", category: 'graphics', css_class: 'graphic'
+        .information
+          %h1.text-h0.mb-12 Content Factotum
+          .details
+            .remote-tag
+              = graphical_icon "stars"
+              100% remote
+            .salary Salary: £50k (~$68k, ~€60k)
+
+  %article
+    .sm-container
+      %p.text-p-2xlarge.mb-40
+        %strong Become Exercism's Content Factotum!
+        We need someone to lead and support our team of content-creating volunteers as we create one of the best educational resources on the planet.
+        Are you looking for a job that's a mixture of strategy, authoring, editing and leading, this might be the perfect role for you?
+        Do you have passion for programming?
+        If so, this role might be perfect for you.
+
+      %h2.text-h2.mb-12 Summary
+      %p.text-p-xlarge.mb-16
+        Exercism is a not-for-profit organisation providing world-class programming education to anyone for free.
+        Our online platform is centred around self-directed learning and volunteer mentoring, and has nearly one million members.
+        Over the next 5 years, we want to become the defacto programming education platform that anyone, anywhere can use for free.
+        Through our work we aim to improve the global standard of programming and increase participation in tech from under-represented groups.
+        You can learn more about Exercism in our #{link_to '"About Exercism" pages', 'https://exercism.org/about', target: '_blank', rel: :noopener}.
+      %p.text-p-xlarge.mb-16
+        At the heart of Exercism is our content - thousands of programming exercises across over 5o programming languages, designed to teach and enable practice of programming concepts, nearly all of which has been written by our volunteer community.
+        As we continue to grow our content, and plan to expand it beyond our traditional exercise format, we need someone to lead our content strategy.
+        This is a hands-on role, where you'll be writing and editor content, alongside planning and leading our volunteer maintainers.
```

Comment:
> you'll be writing and editor content

editing?
> alongside planning and leading our volunteer maintainers.

I parsed this as _alongside (planning and leading) our volunteer maintainers._